### ISSUE TITLE
Masthead:- Accessibility :- Close icon (issues related to hamburger menu once it is expanded)

### ISSUE THREAD

**Issue by chetankan** [carbon-design-system/ibm-dotcom-library#3161]
Issue Description :- There is no label for the mentioned control.

Recommended Fix:- Give appropriate label.

URL : http://ibmdotcom-web-components-canary.mybluemix.net/?

Happening on :- Voiceover ,iPhone

**Comment by ljcarot** [comment]
@chetankan Is this when you tested the Web Components code or on React code too?  Please clarify when you open bug reports so we know where to address.

**Comment by chetankan** [comment]
@ljcarot : In web component, added url now.

**Comment by RaphaelAmadeu** [comment]
As the `carbon-custom-elements`  were updated, their close button now  have a proper aria-label
